feature-utils/poly-look/src/react-components/overlays/sideSheet.css

```diff
@@ -0,0 +1,34 @@
+.poly-side-sheet {
+  max-width: 288px;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-gray-100);
+  color: var(--color-dark);
+  z-index: 100;
+  overflow: hidden scroll;
+  padding: 18px 16px 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+}
+.poly-side-sheet .header {
+  display: flex;
+  align-items: center;
+  min-height: 48px;
+  margin-bottom: 23px;
+}
+.poly-side-sheet .close-button-left {
+  position: absolute;
+  left: 0;
+}
+.poly-side-sheet .title {
+  font-size: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+}
+.poly-side-sheet .button-holder {
```

Comment:
Is there anything in [`themes.css`](https://github.com/polypoly-eu/polyPod/blob/5345dce39655e5ca86566c94192dfc48f4f0d879/feature-utils/poly-look/src/css/themes.css) that could be reused here?
Maybe linking in a comment the figma designs would also help.

Reply:
Don't we normally link to those in StoryBook? Don't see it in this PR though, and too lazy to check the surrounding code at this hour :D